mc-sema/cfgToLLVM/x86Instrs_Branches.cpp

```diff
@@ -1021,57 +842,23 @@ static InstTransResult translate_CALL64r(NativeModulePtr natM,
   TASSERT(tgtOp.isReg(), "");
 
   //read the register
-  Value *fromReg = R_READ<64>(block, tgtOp.getReg());
+  Value *fromReg = R_READ<width>(block, tgtOp.getReg());
 
   Module *M = block->getParent()->getParent();
   const std::string &triple = M->getTargetTriple();
   if (triple != WINDOWS_TRIPLE) {
-    x86_64::writeFakeReturnAddr(block);
+      if(width == 64) {
```

Comment:
hack: can you use this slot to store useful info? E.g. pointer to state structure, current recursion depth, etc.?